### PR TITLE
Add dark mode support to default implementation

### PR DESCRIPTION
**Pull request by lojals**
#### What does this PR do? Any background context you want to provide?
This PR adds Dark Mode support to the default implementation of WLEmptyState
#### Where should the reviewer start?
* `WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift``
* `WLEmptyState/Classes/Views/EmptyStateView.swift`
#### How should this be manually tested?
Turn on and off dark mode on the Example app.
#### Screenshots (if appropriate)
Support for Dark mode
| Dark mode | Light mode |
| ------------- | ------------- |
| simulator_screenshot_C4B716C0-3252-4C27-9535-279667489552 |  Simulator Screen Shot - iPod touch (7th generation) - 2020-11-05 at 21 41 12  |

### CODE REVIEW

File: WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import UIKit
+
+extension UIColor {
+
+    static var accent: UIColor = {
+        return UIColor { (UITraitCollection: UITraitCollection) -> UIColor in
+            if UITraitCollection.userInterfaceStyle == .dark {
+                return UIColor.white
+            } else {
+                return UIColor.black
```
Comment (alexandercg) [inline WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift:18]: could it be like below?
```suggestion
                return .white
            } else {
                return .black
```

File: WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//  Created by Jorge Ovalle on 09/11/20.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+
+    static var accent: UIColor = {
+        return UIColor { (UITraitCollection: UITraitCollection) -> UIColor in
```
Comment (alexandercg) [inline WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift:14]: I don’t have strong opinions on this but can be like
```suggestion
        return UIColor { traitCollection in
```

**Review by alexandercg — APPROVED** [review]
LGTM, just left minor comments

File: Example/WLEmptyState/UIColor+Example.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+extension UIColor {
+
+    static var accent: UIColor = {
+        return UIColor { (UITraitCollection: UITraitCollection) -> UIColor in
+            if UITraitCollection.userInterfaceStyle == .dark {
+                return UIColor.white
+            } else {
+                return UIColor.black
+            }
```
Comment (saulurias) [inline Example/WLEmptyState/UIColor+Example.swift:19]: Maybe
```suggestion
          return (UITraitCollection.userInterfaceStyle == .dark) ? .white : .black
```

File: WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift [hunk 4]
```diff
@@ -0,0 +1,23 @@
+//
+//  UIColor+WLEmptyState.swift
+//  WLEmptyState
+//
+//  Created by Jorge Ovalle on 09/11/20.
+//
+
+import Foundation
```
Comment (saulurias) [inline WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift:8]:
```suggestion
```

**Review by saulurias — APPROVED** [review]
Just left minor suggestions, but in general, looks good to me.

File: WLEmptyState/Classes/Protocols/WLEmptyStateDataSource.swift [hunk 5]
```diff
@@ -40,9 +40,9 @@ public extension WLEmptyStateDataSource {
     
     func imageForEmptyDataSet() -> UIImage? {
         guard let url = (Bundle(for: WLEmptyState.self)).url(forResource: "WLEmptyState", withExtension: "bundle"),
-            let bundle = Bundle(url: url) else { return nil }
-        let image = UIImage(named: EmptyStateView.DefaultConstants.image, in: bundle, compatibleWith: nil)
-        return image
+            let resourceBundle = Bundle(url: url) else { return nil }
+        let image = UIImage(named: EmptyStateView.DefaultConstants.image, in: resourceBundle, compatibleWith: nil)
+        return  image?.withTintColor(.accent)
```
Comment (leonardo-garcia) [inline WLEmptyState/Classes/Protocols/WLEmptyStateDataSource.swift:45]: extra space before image I guess 🤔

**Pull request merged**